common/src/main/java/org/apache/nemo/common/DirectByteArrayOutputStream.java

```diff
@@ -42,16 +43,11 @@ public DirectByteArrayOutputStream(final int size) {
   }
 
   /**
+   * Note that serializedBytes include invalid bytes.
+   * So we have to use it with the actualLength whenever needed
```

Comment:
How can the user know it's "actualLength"?

Reply:
It can be obtained by using .size() method. Would it be better if I change the comment, from actualLength to size() ?

Reply:
@sanha It was obtained by `getCount()` method in `DirectByteArrayOutputStream` before, but it turns out that `getCount()` is no different from `size()` except that `size()` is wrapped with `synchronized`. So, using `getCount()` to figure out `actualLength` is no longer necessary.

@hy00nc +1 for adding a comment that the `actualLength` could be get from `.size()` method.

Reply:
Understood. Thanks for your explanation! I'm okay with this PR.

Reply:
@sanha Thank you very much. @hy00nc I will merge this PR after the comment is modified.

Reply:
@DifferentSC Added them just now. Thank you!